MMM-COVID19.js

```diff
@@ -87,39 +88,51 @@ Module.register("MMM-COVID19", {
 
     // header row
     var headerRow = document.createElement("tr"),
-        headerconfirmedCell = document.createElement("td"),
+        headerConfirmedCell = document.createElement("td"),
+        headerCasesPerMCell = document.createElement("td"),
         headerNewConfirmedCell = document.createElement("td"),
         headerCountryNameCell = document.createElement("td"),
-        headerrecoveredCell = document.createElement("td"),
-        headerdeathsCell = document.createElement("td"),
+        headerRecoveredCell = document.createElement("td"),
+        headerDeathsCell = document.createElement("td"),
         headerNewDeathsCell = document.createElement("td"),
-        headeractiveCell = document.createElement("td")
+        headerSeriousCell = document.createElement("td"),
+        headerActiveCell = document.createElement("td");
 
     headerCountryNameCell.innerHTML = ''
-    headerconfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
-    headerconfirmedCell.innerHTML = this.translate('Confirmed')
+    headerConfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
+    headerConfirmedCell.innerHTML = this.translate('Confirmed')
+    headerCasesPerMCell.className = 'number per M ' + this.config.headerRowClass
```

Comment:
@testdruif wrong className used here
Please use `number` only

Reply:
Just as a question. Following this request/guideline, should "number confirmed " also just be "number "? Or am I missing something major in my thought pattern?

I don't see a separate entry for confirmed in css

Reply:
yes, I added the class names at the js if in the future I would like to change the colors or add some specific css to it.